dags/mozaggregator_prerelease_new.py

```diff
@@ -0,0 +1,185 @@
+import json
+import os
+from datetime import datetime, timedelta
+
+from airflow import DAG
+from airflow.contrib.operators.gcs_delete_operator import GoogleCloudStorageDeleteOperator
+from airflow.contrib.hooks.gcp_api_base_hook import GoogleCloudBaseHook
+from airflow.operators.subdag_operator import SubDagOperator
+from utils.dataproc import moz_dataproc_pyspark_runner, copy_artifacts_dev
+from utils.gcp import gke_command
+
+# TODO - should i disable this? since this job will only backfill the lean database
+# and then once we're caught up, I simply plan to switch DNS to point the production mozaggregator to the new trimmed database
+# (will also redeploy mozagg prod to get rid of datadog)
+EXPORT_TO_AVRO = True
+
+default_args = {
+    "owner": "frank@mozilla.com",
+    "depends_on_past": True,
+    # TODO - double check? our last run before snapshot was for runid 2/19/2020 but started 2/20/2020 0 UTC
+    # or 2/19/2020 at 4pm PST
+    "start_date": datetime(2020, 02, 20),
```

Comment:
Anthony, was the last date you saw when checking the data 2/19?

Reply:
Yes, `20200219` is the last date seen.